### PR TITLE
Devoms

### PR DESCRIPTION
**Pull request by Stijnc**
### Contributing guide

https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md
### Changelog
- replaced  left and right parenthesis with a '-'  in oms-all-deploy/nestedtemplates/omsWorkspace.json
- replaced  left and right parenthesis with a '-' in oms-existing-storage-account/azuredeploy.json
- replaced  left and right parenthesis with a '-' in oms-service-fabric-solution/azuredeploy.json
### Description of the change

replaced the left and right parenthesis with a '-' as described in #2491

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@Stijnc__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by Stijnc** [comment]
apparently the parameternames in the parameter file for oms-existing-storage-account do not match with the parameters specified in azuredeploy.json.

This should be fixed as well now

**Comment by azurecla** [comment]
__@Stijnc__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

File: oms-all-deploy/nestedtemplates/omsWorkspace.json [hunk 1]
```diff
@@ -179,7 +179,7 @@
           },
           "properties": {
             "ETag": "*",
-            "query": "Type=Event EventID=40 OR EventID=36� Source=\"DISK\" EventLog=\"System\" TimeGenerated>NOW-24HOURS | Measure Count() By Compute",
+            "query": "Type=Event EventID=40 OR EventID=36� Source=\"DISK\" EventLog=\"System\" TimeGenerated>NOW-24HOURS | Measure Count() By Compute",
```
Comment (gatneil) [inline oms-all-deploy/nestedtemplates/omsWorkspace.json]: This renders as a non-displayable character in the PR UI; what character is this?

Comment (gatneil) [inline oms-all-deploy/nestedtemplates/omsWorkspace.json]: This displays as having a non-displayable character after "36". What character is it?

**Comment by gatneil** [comment]
No activity; closing

**Pull request closed without merging**